### ISSUE TITLE
All Selected Sheets in generateMultiSheets

### ISSUE THREAD

**Issue by dhiegosad** [functionscope/Node-Excel-Export#43]
Hi,

I  am facing an issue when I am generating multisheets. I do not why, but **all my sheets** are selected previously. 

So... My question is: the library provides a property that I must to set in order to not select all worksheets or I found a bug?

If it was a bug, maybe we set to [tabSelected to '0' in sheetFront](https://github.com/functionscope/Node-Excel-Export/blob/master/sheet.js#L2). What do you think?

**Comment by himanshuj89** [comment]
Yeah..I faced this issue as well. As suggested by @dhiegosad  setting tabSelected to '0' solves this. 

thank you.
